Examples/Modules/Chapter 12/Ex12_17/RandomBoxes.cppm

```diff
@@ -0,0 +1,26 @@
+export module box.random;
+import box;
```

Comment:
You don't want to export box?

Reply:
I could. But I didn't in the book, so let's leave it...